tests/test_games_do_not_fail.py

```diff
@@ -104,6 +104,11 @@ def test_language_bottleneck():
         "egg.zoo.language_bottleneck.guess_number.train", dict(n_epochs=1, vocab_size=5)
     )
 
+def test_emcom_as_ssl_cifar():
+    run_game(
+        "egg.zoo.emcom_as_ssl.train", dict(dataset_name='cifar10', n_epoch=1)
+    )
+
```

Comment:
Have you tried this locally? It seems CI are failing with a timeout error (> 10mins). Maybe defaults are creating a big (image_size too large and hidden dims unnecessary big?) game?